source/blog/2020-11-03-solidusconf-2020-recap.html.markdown

```diff
@@ -0,0 +1,112 @@
+---
+title: "SolidusConf 2020 Recap"
+date: 2020-11-03
+tags: Solidus
+author: Sean Denny
+cover_image: /blog/2020/07/17/announcing-solidusconf-2020/cover.jpg
+description: SolidusConf 2020 has concluded! Let's take a look back at some of the speakers and what we learned.
+---
+
+SolidusConf 2020 is over! Wow, what a conference! First of all, I want to say thank you to everyone who attended and supported the conference. It's so great to see the community come together, <s>even</s> especially in these hard times!
+
+Each one of the talks was excellent, and I’m glad that we were able to hear from such talented people! Here’s a rundown of each of the talks and events that occurred during the conference:
+
+##Day 1
+
+###Peter Berkenbosch
+######Building Payment Provider Integrations
+
+[![Peter Berkenbosch - Building Payment Provider Integrations | SolidusConf 2020](/blog/2020-09-04-solidus-conf-2020-presenter-peter-berkenbosch/peter.jpg)](http://www.youtube.com/watch?v=pWznleqof8s "Peter Berkenbosch - Building Payment Provider Integrations | SolidusConf 2020")
```

Comment:
Oh, I thought initially that that didn't work - at least, the sources I saw said that you couldn't use embeded videos in markdown, but I just tried it and it looks like it works, so that's what I'll do!

Reply:
Added, thanks for the suggestion!